Document qlight CLI options (#212)
ConsenSys/doc.goquorum: docs/configure-and-manage/configure/qlight-node.md

> # Configure a GoQuorum qlight node

Reviewer: Same comment as the previous one. I suggest the simple following title as the page has sections for node and client:
```suggestion
# Configure GoQuorum qlight
```
And it's a whole system in the end as you don't use client without node or node without client. So simply talking about "GoQuorum qlight" implies both node and client and their interaction. Just a suggestion though.

# Configure a GoQuorum qlight node

A [qlight client node](../../concepts/qlight-node.md) requires a full node configured to act as a qlight server.
The server node is usually set up to support [multiple private states](../../concepts/multi-tenancy.md#multiple-private-states)
(MPS), with the qlight client set up to use a [private state identifier](../../concepts/multi-tenancy.md#private-state-identifier)
(PSI) which is managed by the server node.

!!! note

    Setting up a server node to support MPS is not required, for example, when setting up a qlight client for offloading processing from a full node.

## Configure qlight client

Configure the qlight client using the [`--qlight.client.*`](../../reference/cli-syntax.md#qlightclient) command line options.

If the server node has the RPC API secured using TLS, then you must set the [`--qlight.client.rpc.tls.*`](../../reference/cli-syntax.md#qlightclientrpctls) command line options.

> If the qlight client node is halted, on restart it resyncs with any blocks that were missed when it was not running.

Reviewer 2: ```suggestion
If you stop the qlight client node, on restart it resyncs with any blocks it missed when it was not running.
```

> If the qlight client node is halted, on restart it resyncs with any blocks that were missed when it was not running.

Reviewer 2: ```suggestion
If you stop the qlight client node, on restart it resyncs with any blocks it missed when it was not running.
```

## Configure server node

Configure the qlight server using the [`--qlight.server.*`](../../reference/cli-syntax.md#qlightserver) command line options.

### Network IP restriction

This restricts communication to specified IP networks (CIDR masks).
Specify the network mask on the qlight server using [`--qlight.server.p2p.netrestrict`](../../reference/cli-syntax.md#qlightserverp2pnetrestrict).

### File based permissioning

> File based permissioning allows qlight clients to be checked against a permissioned list and a disallowed list.

Reviewer 2: ```suggestion
File based permissioning allows you to check qlight clients against a permissioned list and a disallowed list.
```

Enable file based permissioning on the server node using [`--qlight.server.p2p.permissioning`](../../reference/cli-syntax.md#qlightserverp2ppermissioning).

The default files are `permissioned-nodes.json` and `disallowed-nodes.json`.

> However, you can specify a file prefix using [`--qlight.server.p2p.permissioning.prefix`](../../reference/cli-syntax.md#qlightserverp2ppermissioningprefix),
> in which case the filename is: the prefix, followed by a hyphen, followed by the default file name.

Reviewer 2: ```suggestion
You can specify a file prefix using [`--qlight.server.p2p.permissioning.prefix`](../../reference/cli-syntax.md#qlightserverp2ppermissioningprefix).
```
